src/hooks/coreSDK/__tests__/safeCoreSDK.test.ts

```diff
@@ -137,16 +144,17 @@ describe('safeCoreSDK', () => {
           implementationVersionState: ImplementationVersionState.UP_TO_DATE,
         })
 
-        expect(sdk).toBeInstanceOf(Safe)
```

Comment:
Had to change these tests because we can't create an actual instance of the SDK anymore. Basically, the protocol-kit creates a provider internally now that we can't mock so real RPC requests would happen. Instead we can check that the SDK was called with the parameters we expect.